<?xml version="1.0"?>

<launch>
    <arg name="max_vel_linear"    default="1.0"/>
    <arg name="max_vel_angular"   default="1.0"/>
    <arg name="linear_vel_step"   default="0.1"/>
    <arg name="angular_vel_step"  default="0.1"/>
    <arg name="cmd_vel"           default="cmd_vel"/>


    <!-- Node for keyboard operation -->
    <node pkg="tsukuba2022" type="teleop_key.py" name="teleop_key" output="screen">
        <remap from="cmd_vel"  to="$(arg cmd_vel)"/>
        <param name="max_vel_linear"    value="$(arg max_vel_linear)"/>
        <param name="max_vel_angular"   value="$(arg max_vel_angular)"/>
        <param name="linear_vel_step"   value="$(arg linear_vel_step)"/>
        <param name="angular_vel_step"  value="$(arg angular_vel_step)"/>
    </node>

</launch>